### PR TITLE
[improvement-13701][dolphinscheduler-api] Add Unit Test for DynamicTaskTypeController

### PR DESCRIPTION
**Pull request by jiangyq7**
Add DynamicTaskTypeControllerTest

### CODE REVIEW

**Comment by jiangyq7** [comment]
@SbloodyS Hello,Could you help me to add version or label 3.2.0?

**Comment by jiangyq7** [comment]
@caishunfeng Hello,Could you please assign this to me,and add a version number 3.2.0 for this PR?

**Comment by jiangyq7** [comment]
Could you help me to create an Lable or version number? version number is 3.2.0 @SbloodyS

**Comment by fuchanghai** [comment]
@jiangyq7 plz link this PR to you issue

File: dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/controller/DynamicTaskTypeControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package org.apache.dolphinscheduler.api.controller;
+
+
+import org.apache.dolphinscheduler.api.enums.Status;
+import org.apache.dolphinscheduler.api.utils.Result;
+import org.apache.dolphinscheduler.common.utils.JSONUtils;
+import org.junit.jupiter.api.*;
```
Comment (EricGao888) [inline dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/controller/DynamicTaskTypeControllerTest.java:7]: Please avoid wildcard imports here.

File: dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/controller/DynamicTaskTypeControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+import java.util.HashMap;
+import java.util.List;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * Dynamic Task controller test
```
Comment (EricGao888) [inline dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/controller/DynamicTaskTypeControllerTest.java:21]: This comment is unnecessary. Everybody knows it is test for dynamic controller.

File: dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/controller/DynamicTaskTypeControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    public void initSetUp() {
+        setUp();
+    }
+
+    @AfterEach
+    public void afterEach() throws Exception {
+        after();
+    }
+
+    @Disabled("Query TaskCategories")
```
Comment (EricGao888) [inline dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/controller/DynamicTaskTypeControllerTest.java:36]: Why disabled?

**Pull request closed without merging**